### PR TITLE
[fix](move-memtable) pass load stream num to backends

### PR DESCRIPTION
**Pull request by kaijchen**
## Proposed changes

Some sinks may close load before other sinks open the stream, fix it by passing num total streams to downstream BE.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by kaijchen** [comment]
run buildall

File: be/src/runtime/load_stream.cpp [hunk 1]
```diff
@@ -258,19 +259,21 @@ Status LoadStream::init(const POpenStreamSinkRequest* request) {
 Status LoadStream::close(int64_t src_id, const std::vector<PTabletID>& tablets_to_commit,
                          std::vector<int64_t>* success_tablet_ids,
                          std::vector<int64_t>* failed_tablet_ids) {
-    std::lock_guard lock_guard(_lock);
+    std::lock_guard<bthread::Mutex> lock_guard(_lock);
     SCOPED_TIMER(_close_wait_timer);
 
     // we do nothing until recv CLOSE_LOAD from all stream to ensure all data are handled before ack
-    _open_streams[src_id]--;
+    _close_load_cnt++;
```
Comment (dataroaring) [inline be/src/runtime/load_stream.cpp:266]: member of LoadStream,std::set LoadStream::_tablets_to_commit;
_tablets_to_commit.add(tablets_to_commit);

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by Hastyshell — APPROVED** [review]
LGTM

**Pull request merged**